src/ngraph/runtime/cpu/cpu_emitter.cpp

```diff
@@ -2511,6 +2511,11 @@ namespace ngraph
                     auto data_format = mkldnn_utils::get_input_mkldnn_format(node, 0);
                     auto weights_format = mkldnn_utils::get_input_mkldnn_format(node, 1);
                     auto bias_format = mkldnn_utils::get_input_mkldnn_format(node, 2);
+                    // HACK to help MKLDNN pick the right implementation
+                    if (weights_format == mkldnn::memory::format::nchw)
```

Comment:
should we just move this hack into `get_input_mkldnn_format` which would apply the  hack if op is a `Convolution`?